contracts/lib/Structs.sol

```diff
@@ -3,18 +3,20 @@ pragma solidity ^0.8.0;
 
 library Structs {
     struct Vote {
-        uint256 value;
-        uint256 weight;
+        uint32 epoch;
+        uint32[] values;
     }
 
+    struct Commitment {
+        uint32 epoch;
+        bytes32 commitmentHash;
+    }
     struct Staker {
-        uint256 id;
+        uint32 id;
         address _address;
         uint256 stake;
-        uint256 age;
-        uint256 epochStaked;
-        uint256 epochLastCommitted;
-        uint256 epochLastRevealed;
+        uint32 age;
```

Comment:
We should move this to the top with id 
like 
uint32 id;
uint32 age;
unit32 epochStaked

To allow tight packing: https://fravoll.github.io/solidity-patterns/tight_variable_packing.html
This way all three would be in one storage slot 
saving storage and interaction cost

Reply:
I tried it. doesnt work. Seems optimizer does it automatically.

Reply:
oh okay then